proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -130,6 +130,37 @@ service Microgrid {
     };
   }
 
+  // Adds inclusion bounds for a given metric of a given component, and returns
+  // the UTC timestamp until which the given inclusion bounds will stay in
+  // effect.
```

Comment:
I'm not sure its wrong but maybe it would read better as:

```
Adds inclusion bounds for a given metric of a given component. Returns the time at which the bounds will expire (as a timestamp (UTC)) 
```

Reply:
Done